### DISCUSSION TITLE
每日一槽

### DISCUSSION THREAD

**Discussion by Huaijin2333** [share121/inter-knot#857] [其他]
王*启真是个狗，不干人事，天天炉管

**Comment by WLMRMZ** [comment]
在绳网不能人身攻击哦，这里不是法外之地，会被治安局抓走的！

**Reply by Dream-Alan** [reply]
绳网官方温馨提醒您：绳网不是法外之地，请注意您的一言一行

**Comment by keerdun** [comment]
绳网不能人身攻击啊喂！

**Comment by Lephaethon** [comment]
绳网禁止人身攻击 @share121

**Comment by wwwwsn147** [comment]
随便侮辱 谩骂他人是不对的哦 @shair121

**Comment by Lanxey** [comment]
你说的这个“王*启”是不是你自己

**Comment by Qiling-XwX** [comment]
这样是不对的！
![6FB9E1F28B0A759CFB8416545E03A53B](https://github.com/user-attachments/assets/282abf07-30ee-4a3d-913c-738d055a71da)

**Comment by Larksonguesinaspic** [comment]
治安局呢？！
治安局呢？？？？！！！！
![image](https://github.com/user-attachments/assets/eac432a2-499f-4376-ab56-82249a4fb820)

**Comment by lllliufeng** [comment]
呼叫朱鸢探长！！！
